docs/cli/cli-orders.md

```diff
@@ -467,7 +467,23 @@ curl -s https://raw.githubusercontent.com/planetlabs/planet-client-python/main/d
 
 ### Harmonize
 
-TODO
+The harmonize tool allows you to compare data to different generations of satellites by radiometrically harmonizing imagery captured by one satellite instrument type to imagery captured by another. To harmonize your data to a sensor you must define the sensor you wish to harmonize with in your `tools.json`. Currently, only "PS2" (Dove Classic) and "Sentinel-2" are supported as target sensors. The Sentinel-2 target only harmonizes PSScene surface reflectance bundle types (`analytic_8b_sr_udm2`, `analytic_sr_udm2`). The PS2 target only works on analytic bundles from Dove-R (`PS2.SD`).
+
+```json
+[
+  {
+    "harmonize": {
+      "target_sensor": "PS2"
```

Comment:
Thanks for the new docs! 

Could you change this target sensor to be Sentinel-2? That's the much better one for everyone to use. You'd only use PS2 if you had a bunch of legacy dove classic data. 

```suggestion
      "target_sensor": "Sentinel-2"
```

Reply:
(I think this is the right string...)

Reply:
That's a great idea, thanks for the suggestion!